FFXIVClientStructs/FFXIV/Component/GUI/AtkResNode.cs

```diff
@@ -10,6 +10,7 @@ namespace FFXIVClientStructs.FFXIV.Component.GUI;
 [GenerateInterop(isInherited: true)]
 [Inherits<AtkEventTarget>]
 [StructLayout(LayoutKind.Explicit, Size = 0xB0)]
+[VirtualTable("48 89 01 33 C0 48 89 41 18 66 C7 81 ?? ?? ?? ?? ?? ??", 12, 3)]
```

Comment:
This sig points to the instruction after.
```c
.text:000000014062E067 48 8D 05 F2 30 A1 01       lea     rax, vtbl_Component::GUI::AtkResNode
.text:000000014062E06E 48 89 01                   mov     [rcx], rax
.text:000000014062E071 33 C0                      xor     eax, eax
.text:000000014062E073 48 89 41 18                mov     [rcx+18h], rax
.text:000000014062E077 66 C7 81 8E 00 00 00 FF    mov     word ptr [rcx+8Eh], 0FFh
```

```suggestion
[VirtualTable("48 8D 05 ?? ?? ?? ?? ?? ?? ?? 33 C0 48 89 41 ?? 66 C7 81", 3, 3)]
```